File: LO52_2019_KeepIt/TP1/app/src/main/java/com/example/helloworld/MainActivity.java
package com.example.helloworld;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
public Button btnHello;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        btnHello=findViewById(R.id.btnHello);
        btnHello.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent=new Intent(MainActivity.this,HelloActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: LO52_2019_Equal/TP1_AB/app/src/main/java/com/example/tp1_ab/MainActivity/MainActivity.kt
package com.example.tp1_ab.MainActivity

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.Toast
import com.example.tp1_ab.SecondActivity.SecondActivity


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(com.example.tp1_ab.R.layout.activity_main)


        // get reference to button
        val btn_click_me = findViewById(com.example.tp1_ab.R.id.buttonHelloWorld) as Button
        // set on-click listener
        btn_click_me.setOnClickListener {
            val intent = Intent(this, SecondActivity::class.java)
            // start your next activity
            startActivity(intent)
        }

    }



}